tests/plugins/balloonpanel/_helpers/tools.js

```diff
@@ -57,5 +57,72 @@ var balloonTestsTools = {
 			return document.location.protocol + '//' + document.location.hostname + ( document.location.port ? ':' + document.location.port : '' );
 		}
 		return document.location.origin;
+	},
+
+	getAbsoluteRect: function( editor, element ) {
```

Comment:
Somehow I didn't noticed it earlier... 😞  Since you made `_getAbsoluteRect ` in `balloonpanel` plugin available and it is used in the unit tests, this here is no longer needed and should be removed.

Reply:
done ;)